plugin/tick.vim

```diff
@@ -2,7 +2,140 @@
 "
 " Much of this plugin was copied from Fatih's wonderful https://github.com/fatih/vim-go/, so send him your thanks!
 
+" install necessary Go tools
+if exists("g:tick_loaded_install")
+    finish
+endif
+let g:tick_loaded_install = 1
+
+
+" these packages are used by vim-tick and can be automatically installed if
+" needed by the user with TickInstallBinaries
+let s:packages = [
+            \ "github.com/influxdata/kapacitor/tick/cmd/tickfmt"
+            \ ]
+
+" These commands are available on any filetypes
 command! TickFmt call tick#fmt#Format()
+command! TickInstallBinaries call s:TickInstallBinaries(-1)
+command! TickUpdateBinaries call s:TickInstallBinaries(1)
+command! -nargs=? -complete=dir GoPath call go#path#GoPath(<f-args>)
```

Comment:
Well, maybe I was too quick to submit the pull request. It's not a dep on vim-go as much as it is the go binaries. Which means, there probably should be a mechanism to install those binaries in the same way vim-go installs the binaries. My intent was to maintain the same level of assumptions you already were making, such as that the user could execute "go get" on the command line; hence, go binaries would installed. I can do some more fiddling if you'd like. I just like the idea of a self-contained plugin that can manage itself (for the most part).